### PR TITLE
Polish Function.Lift

### PR DESCRIPTION
**Pull request by jkopanski**
There was F-cong missing. Additionally I've copied some utils from felix-boolean, that I think are useful for other that kind of libraries.

### CODE REVIEW

File: src/Felix/Instances/Function/Lift.agda [hunk 1]
```diff
@@ -8,10 +8,24 @@ open Lift
...
+open import Felix.Object
 
 private module F {ℓ} where open import Felix.Instances.Function ℓ public
 open F
 
+private
+  variable
+    A : Set a
+
+lift₀ : ⦃ _ : Products (Set (a ⊔ b)) ⦄ → A → (⊤ F.⇾ Lift b A)
```
Comment (conal) [inline src/Felix/Instances/Function/Lift.agda:20]: Considering the "`open F`" above, do you need the "`F.`" here and below?

**Comment by conal** [comment]
I guess I wasn't using this module and let it get out of sync. Thanks for updating it, @jkopanski!

File: src/Felix/Instances/Function/Lift.agda (hunk 1 above)
Comment (jkopanski) [reply src/Felix/Instances/Function/Lift.agda]: I've missed this comment somehow.

I'm not sure why I did this in the first place, probably copied it from outside felix like so.

**Comment by jkopanski** [comment]
I've used the already opened module, as noted in your comment, would you be open to merging this in? I keep copying those helpers in all felix related code.

Also:
> I wasn't using this module and let it get out of sync.

would it be ok if I contributed some ci setup using github actions? (probably done with nix)

**Comment by conal** [comment]
> I've used the already opened module, as noted in your comment, would you be open to merging this in? I keep copying those helpers in all felix related code.

I've forgotten how I was using this module and why I (apparently) stopped using it. How are you using these operations?

**Comment by conal** [comment]
> would it be ok if I contributed some ci setup using github actions? (probably done with nix)

I'm intrigued. What would the github actions do?

**Comment by jkopanski** [comment]
> I'm intrigued. What would the github actions do?

For agda projects it's mostly running agda on `Everything.agda` (or `Felix/All.agda` here) file, making sure it typechecks. But I've also seen some projects are dynamically generating Everything file, so it will include well, everything ;). It's almost one line shell script.

**Comment by jkopanski** [comment]
> I've forgotten how I was using this module and why I (apparently) stopped using it. How are you using these operations?

The same way, you used it in the [felix-boolean](https://github.com/conal/felix-boolean/blob/main/src/Felix/Boolean/Function.lagda.tex#L39), so lifting `Op`s in `Set` to `Set ℓ`. However just yesterday I was having second thoughts about is this the right way, as I don't see (yet at least) how to lift properties related to those types as well.

The _end goal_ is to define algebraic structures that would play nice with felix. Right now I am taking a look on what was done in denotational-hardware (the `HasAlgebra` module, etc.)

**Comment by conal** [comment]
@jkopanski I just committed https://github.com/conal/felix/commit/3a0fc5f7aba91233d9ec4dd20787756891c7148c, which I seem to have copied from this PR. I don't remember why I didn't just merge this PR instead. I'm closing this PR, but please reopen it or add an issue if I've missed something. Sorry for stealing credit for the changes!

**Pull request closed without merging**